### ISSUE TITLE
Allow customisation of the nav-toggle

### ISSUE THREAD

**Issue by craigmdennis** [viljamis/responsive-nav.js#184]
It would be great to be able to change the classes applied to the nav toggle as well as the main navigation element.

**Comment by kush1960** [comment]
+1 for this feature.

**Comment by samikeijonen** [comment]
You can do both using `customToggle` and `navClass` options.

**Comment by kush1960** [comment]
As far as I can tell, there isn't a way to modify the class applied top the js inserted button used to trigger the menu `<a href="#" class="nav-toggle" aria-hidden="false">Menu</a>`
Isn't `customToggle` for targetting existing html created for the button?
And `navClass` applies a class to the list used for the menu?

I was hoping to replace the class used in the injected a href. Eg. `<a href="#" class="nav-other-nav-toggle" aria-hidden="false">Menu</a>`

**Comment by samikeijonen** [comment]
Yes you should `customToggle` for your own `button`. And note that it really should be a button, not a link. 

> And navClass applies a class to the list used for the menu?

Noup, it's for `nav` itself.

**Comment by kush1960** [comment]
Thanks for clarifying.
